File: C++ solutions/2469. Convert the Temperature.cpp
//approach :1
class Solution {
public:
    vector<double> convertTemperature(double celsius) {
        return {(celsius+273.15),(celsius * 1.80 + 32.00)};
    }
};

//approach :2
class Solution {
public:
    vector<double> convertTemperature(double celsius) {
        vector<double>ans;
        ans.push_back(celsius + 273.15);
        ans.push_back(celsius * 1.80 + 32.00);
        return ans;
    }
};

File: JavaScript solutions/2469. Convert the Temperature.js
//approach : 1
/**
 * @param {number} celsius
 * @return {number[]}
 */
var convertTemperature = function (celsius) {
  let ans = [];
  ans.push(celsius + 273.15);
  ans.push(celsius * 1.8 + 32.0);
  return ans;
};

//approach : 2
/**
 * @param {number} celsius
 * @return {number[]}
 */
var convertTemperature = function (celsius) {
  return [celsius + 273.15, celsius * 1.8 + 32.0];
};
